diff --git a/source/fhir.ini b/source/fhir.ini
index bde93082e56..8ae288b523a 100644
--- a/source/fhir.ini
+++ b/source/fhir.ini
@@ -2028,7 +2028,10 @@ RelatedArtifact.url=-> document
 RequestGroup=@RequestOrchestration
 ResearchDefinition=
 ResearchElementDefinition=
-ResearchStudy.arm=
+ResearchStudy.arm=@ResearchStudy.comparisonGroup
+ResearchStudy.arm.name=@ResearchStudy.comparisonGroup.name
+ResearchStudy.arm.type=
+ResearchStudy.arm.description=@ResearchStudy.comparisonGroup.description
 ResearchStudy.category=@ResearchStudy.studyDesign
 ResearchStudy.contact=-> Use Organization.contact or PractitionerRole.contact for study contact details
 ResearchStudy.enrollment=
