diff --git a/source/fhir.ini b/source/fhir.ini
index 97f0145cee4..8c2b9c7f97e 100644
--- a/source/fhir.ini
+++ b/source/fhir.ini
@@ -2310,7 +2310,7 @@ Group.characteristic.formula=+
 Group.characteristic.instances[x]=+
 Group.characteristic.method=+
 Group.characteristic.offset=+
-Group.characteristic.timing=+
+Group.characteristic.relativeTime=+
 Group.combinationMethod=+
 Group.combinationThreshold=+
 Group.contact=+
