diff --git a/source/fhir.ini b/source/fhir.ini
index 8c2b9c7f97e..013e06bb706 100644
--- a/source/fhir.ini
+++ b/source/fhir.ini
@@ -1933,7 +1933,7 @@ MedicinalProductIndication=-> ClinicalUseDefinition
 MedicinalProductIngredient=-> Ingredient
 MedicinalProductInteraction=-> ClinicalUseDefinition
 MedicinalProductManufactured=-> ManufacturedItemDefinition
-MedicinalProductPackaged=-> ProductPackagedDefinition
+MedicinalProductPackaged=-> PackagedProductDefinition
 MedicinalProductPharmaceutical=-> AdmininistrableProductDefinition
 MedicinalProductUndesirableEffect=-> ClinicalUseDefinition
 MessageHeader.enterer=
