diff --git a/source/oids.ini b/source/oids.ini
index aa3b5360c17..8947b13924f 100644
--- a/source/oids.ini
+++ b/source/oids.ini
@@ -429,7 +429,7 @@ http://hl7.org/fhir/ValueSet/example-filter = 2.16.840.1.113883.4.642.3.3148
 http://hl7.org/fhir/ValueSet/example-intensional = 2.16.840.1.113883.4.642.3.3145
 http://hl7.org/fhir/ValueSet/example-metadata = 2.16.840.1.113883.4.642.3.3139
 http://hl7.org/fhir/ValueSet/example-metadata-2 = 2.16.840.1.113883.4.642.3.3143
-http://hl7.org/fhir/ValueSet/examplescenario-actor-type = 2.16.840.1.113883.4.642.3.858
+http://hl7.org/fhir/ValueSet/actordefinition-actor-type = 2.16.840.1.113883.4.642.3.858
 http://hl7.org/fhir/ValueSet/examplescenario-instance-type = 2.16.840.1.113883.4.642.3.3006
 http://hl7.org/fhir/ValueSet/examplescenario-resource-type = 2.16.840.1.113883.4.642.3.3005
 http://hl7.org/fhir/ValueSet/expansion-parameter-source = 2.16.840.1.113883.4.642.3.1278
@@ -1257,7 +1257,7 @@ http://hl7.org/fhir/ex-pharmaservice = 2.16.840.1.113883.4.642.4.563
 http://hl7.org/fhir/ex-servicemodifier = 2.16.840.1.113883.4.642.4.573
 http://hl7.org/fhir/ex-serviceproduct = 2.16.840.1.113883.4.642.4.561
 http://hl7.org/fhir/ex-udi = 2.16.840.1.113883.4.642.4.555
-http://hl7.org/fhir/examplescenario-actor-type = 2.16.840.1.113883.4.642.4.859
+http://hl7.org/fhir/actordefinition-actor-type = 2.16.840.1.113883.4.642.4.859
 http://hl7.org/fhir/expansion-parameter-source = 2.16.840.1.113883.4.642.4.2015
 http://hl7.org/fhir/expansion-processing-rule = 2.16.840.1.113883.4.642.4.2014
 http://hl7.org/fhir/explanationofbenefit-status = 2.16.840.1.113883.4.642.4.619
