diff --git a/source/fhir.ini b/source/fhir.ini
index 013e06bb706..ef4c9613921 100644
--- a/source/fhir.ini
+++ b/source/fhir.ini
@@ -2679,7 +2679,7 @@ ImmunizationRecommendation.recommendation.protocol.description=ImmunizationRecom
 ImmunizationRecommendation.recommendation.protocol.series=ImmunizationRecommendation.recommendation.series
 ImmunizationRecommendation.recommendation.protocol.doseSequence=ImmunizationRecommendation.recommendation.seriesDoses[x]
 Immunization.notGiven=Immunization.statusReason
-VisionPrescription.dispense.base=VisionPrescription.lensSpecification.base
+VisionPrescription.dispense.base=VisionPrescription.lensSpecification.prism.base
 Claim.information=Claim.supportingInfo
 Claim.item.careTeamLinkId=Claim.item.careTeamSequence
 Claim.item.diagnosisLinkId=Claim.item.diagnosisSequence
