diff --git a/images/source/QuestionnaireItems.drawio b/images/source/QuestionnaireItems.drawio
index b3ca17ddf3e..3077fa3215d 100644
--- a/images/source/QuestionnaireItems.drawio
+++ b/images/source/QuestionnaireItems.drawio
@@ -167,7 +167,7 @@
         <mxCell id="CBZNjNMLxmk2BXDKTw0N-47" value="&lt;span&gt;grp3&lt;/span&gt;" style="whiteSpace=wrap;html=1;rounded=1;shadow=1;labelBackgroundColor=none;strokeWidth=2;fontFamily=Verdana;fontSize=12;align=center;fillColor=#dae8fc;strokeColor=#000000;" parent="1" vertex="1">
           <mxGeometry x="675" y="620" width="50" height="30" as="geometry" />
         </mxCell>
-        <mxCell id="CBZNjNMLxmk2BXDKTw0N-48" value="&lt;span&gt;Q4&lt;br&gt;&lt;/span&gt;" style="whiteSpace=wrap;html=1;rounded=0;shadow=1;labelBackgroundColor=none;strokeWidth=2;fontFamily=Verdana;fontSize=12;align=center;fillColor=#d5e8d4;strokeColor=#000000;" parent="1" vertex="1">
+        <mxCell id="CBZNjNMLxmk2BXDKTw0N-48" value="&lt;span&gt;Q5&lt;br&gt;&lt;/span&gt;" style="whiteSpace=wrap;html=1;rounded=0;shadow=1;labelBackgroundColor=none;strokeWidth=2;fontFamily=Verdana;fontSize=12;align=center;fillColor=#d5e8d4;strokeColor=#000000;" parent="1" vertex="1">
           <mxGeometry x="675" y="680" width="50" height="30" as="geometry" />
         </mxCell>
         <mxCell id="CBZNjNMLxmk2BXDKTw0N-49" value="" style="rounded=0;html=1;labelBackgroundColor=none;startArrow=none;startFill=0;startSize=5;endArrow=none;endFill=0;endSize=5;jettySize=auto;orthogonalLoop=1;strokeWidth=1;fontFamily=Verdana;fontSize=12;entryX=1;entryY=1;entryDx=0;entryDy=0;exitX=0.5;exitY=0;exitDx=0;exitDy=0;" parent="1" source="CBZNjNMLxmk2BXDKTw0N-47" target="CBZNjNMLxmk2BXDKTw0N-60" edge="1">
@@ -219,7 +219,7 @@
         <mxCell id="CBZNjNMLxmk2BXDKTw0N-63" value="&lt;span&gt;grp3&lt;/span&gt;" style="whiteSpace=wrap;html=1;rounded=1;shadow=1;labelBackgroundColor=none;strokeWidth=2;fontFamily=Verdana;fontSize=12;align=center;fillColor=#dae8fc;strokeColor=#000000;" parent="1" vertex="1">
           <mxGeometry x="815" y="620" width="50" height="30" as="geometry" />
         </mxCell>
-        <mxCell id="CBZNjNMLxmk2BXDKTw0N-64" value="&lt;span&gt;Q4&lt;br&gt;&lt;/span&gt;" style="whiteSpace=wrap;html=1;rounded=0;shadow=1;labelBackgroundColor=none;strokeWidth=2;fontFamily=Verdana;fontSize=12;align=center;fillColor=#d5e8d4;strokeColor=#82b366;" parent="1" vertex="1">
+        <mxCell id="CBZNjNMLxmk2BXDKTw0N-64" value="&lt;span&gt;Q5&lt;br&gt;&lt;/span&gt;" style="whiteSpace=wrap;html=1;rounded=0;shadow=1;labelBackgroundColor=none;strokeWidth=2;fontFamily=Verdana;fontSize=12;align=center;fillColor=#d5e8d4;strokeColor=#82b366;" parent="1" vertex="1">
           <mxGeometry x="815" y="680" width="50" height="30" as="geometry" />
         </mxCell>
         <mxCell id="CBZNjNMLxmk2BXDKTw0N-65" value="" style="rounded=0;html=1;labelBackgroundColor=none;startArrow=none;startFill=0;startSize=5;endArrow=none;endFill=0;endSize=5;jettySize=auto;orthogonalLoop=1;strokeWidth=1;fontFamily=Verdana;fontSize=12;entryX=0;entryY=1;entryDx=0;entryDy=0;exitX=0.5;exitY=0;exitDx=0;exitDy=0;" parent="1" source="CBZNjNMLxmk2BXDKTw0N-62" target="CBZNjNMLxmk2BXDKTw0N-61" edge="1">
@@ -266,9 +266,6 @@
         <mxCell id="CBZNjNMLxmk2BXDKTw0N-75" value="Required = true" style="text;html=1;strokeColor=none;fillColor=none;align=left;verticalAlign=middle;whiteSpace=wrap;rounded=0;" parent="1" vertex="1">
           <mxGeometry x="840" y="357" width="90" height="30" as="geometry" />
         </mxCell>
-        <mxCell id="CBZNjNMLxmk2BXDKTw0N-81" value="&lt;span&gt;Q5&lt;br&gt;&lt;/span&gt;" style="whiteSpace=wrap;html=1;rounded=0;shadow=1;labelBackgroundColor=none;strokeWidth=2;fontFamily=Verdana;fontSize=12;align=center;fillColor=#d5e8d4;strokeColor=#000000;" parent="1" vertex="1">
-          <mxGeometry x="815" y="680" width="50" height="30" as="geometry" />
-        </mxCell>
         <mxCell id="CBZNjNMLxmk2BXDKTw0N-82" value="grp1" style="whiteSpace=wrap;html=1;rounded=1;shadow=1;labelBackgroundColor=none;strokeWidth=2;fontFamily=Verdana;fontSize=12;align=center;fillColor=#dae8fc;strokeColor=#000000;" parent="1" vertex="1">
           <mxGeometry x="884" y="455" width="50" height="30" as="geometry" />
         </mxCell>
@@ -324,7 +321,7 @@
             <mxPoint x="710" y="720" as="targetPoint" />
           </mxGeometry>
         </mxCell>
-        <mxCell id="CBZNjNMLxmk2BXDKTw0N-101" value="" style="rounded=0;html=1;labelBackgroundColor=none;startArrow=none;startFill=0;startSize=5;endArrow=none;endFill=0;endSize=5;jettySize=auto;orthogonalLoop=1;strokeWidth=1;fontFamily=Verdana;fontSize=12;entryX=0.5;entryY=1;entryDx=0;entryDy=0;exitX=0.5;exitY=0;exitDx=0;exitDy=0;" parent="1" source="CBZNjNMLxmk2BXDKTw0N-99" target="CBZNjNMLxmk2BXDKTw0N-81" edge="1">
+        <mxCell id="CBZNjNMLxmk2BXDKTw0N-101" value="" style="rounded=0;html=1;labelBackgroundColor=none;startArrow=none;startFill=0;startSize=5;endArrow=none;endFill=0;endSize=5;jettySize=auto;orthogonalLoop=1;strokeWidth=1;fontFamily=Verdana;fontSize=12;entryX=0.5;entryY=1;entryDx=0;entryDy=0;exitX=0.5;exitY=0;exitDx=0;exitDy=0;" parent="1" source="CBZNjNMLxmk2BXDKTw0N-99" target="CBZNjNMLxmk2BXDKTw0N-64" edge="1">
           <mxGeometry x="0.1373" y="16" relative="1" as="geometry">
             <mxPoint as="offset" />
             <mxPoint x="780" y="690" as="sourcePoint" />
