diff --git a/source/activitydefinition/activitydefinition-map-Appointment.fhirmap b/source/activitydefinition/activitydefinition-map-Appointment.fhirmap
index 649136a766e..181b7e74d6b 100644
--- a/source/activitydefinition/activitydefinition-map-Appointment.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-Appointment.fhirmap
@@ -6,7 +6,7 @@ uses "http://hl7.org/fhir/StructureDefinition/Appointment" as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: Appointment as target
 
     "subject" : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.participant as participant then {
@@ -40,7 +40,7 @@ endgroup
 
 group createActDef
     input src: ActivityDefinition as source
-    input tgt: ServiceRequest as target
+    input tgt: Appointment as target
 
 //    "refAd"        : for src.url      as vvv make tgt.basedOn = vvv
 //    "ad.code"      : for src.code     as vvv make tgt.code = vvv          // could be serviceCategory or specialty --> unclear
@@ -77,4 +77,3 @@ group createActDef
     "created"      : for src                 make tgt.created = evaluate( src, now() )
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-map-AppointmentResponse.fhirmap b/source/activitydefinition/activitydefinition-map-AppointmentResponse.fhirmap
index c4638ba1058..181f8b9c660 100644
--- a/source/activitydefinition/activitydefinition-map-AppointmentResponse.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-AppointmentResponse.fhirmap
@@ -6,7 +6,7 @@ uses "http://hl7.org/fhir/StructureDefinition/AppointmentResponse" as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: AppointmentResponse as target
 
 //    "subject" : for src.parameter as p where name='subject' then {
 //        do: for p.value as sbj make tgt.subject = sbj
@@ -38,7 +38,7 @@ endgroup
 
 group createActDef
     input src: ActivityDefinition as source
-    input tgt: ServiceRequest as target
+    input tgt: AppointmentResponse as target
 
 //    "refAd"        : for src.url      as vvv make tgt.instantiatesCanonical = vvv
     "status"       : for src                 make tgt.participantStatus = "needs-action"
@@ -52,4 +52,3 @@ group createActDef
 //    "ad.dosage"    : for src          as vvv check dosage.empty()
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-map-CarePlan.fhirmap b/source/activitydefinition/activitydefinition-map-CarePlan.fhirmap
index 19ab4ee8d58..93a6b2112c8 100644
--- a/source/activitydefinition/activitydefinition-map-CarePlan.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-CarePlan.fhirmap
@@ -6,7 +6,7 @@ uses "http://hl7.org/fhir/StructureDefinition/CarePlan" as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: CarePlan as target
 
     "subject" : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.subject = sbj
@@ -36,7 +36,7 @@ endgroup
 
 group createActDef
     input src: ActivityDefinition as source
-    input tgt: ServiceRequest as target
+    input tgt: CarePlan as target
 
     "ad.code"      : for src.code     as vvv  make tgt.category = vvv
 //    "ad.priority"  : for src.priority as prio make tgt.priority = vvv
@@ -54,4 +54,3 @@ group createActDef
     "created"      : for src                 make tgt.created = evaluate( src, now() )
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-map-Claim.fhirmap b/source/activitydefinition/activitydefinition-map-Claim.fhirmap
index 969bac64a04..be55366b0bc 100644
--- a/source/activitydefinition/activitydefinition-map-Claim.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-Claim.fhirmap
@@ -6,7 +6,7 @@ uses "http://hl7.org/fhir/StructureDefinition/Claim" as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: Claim as target
 
     "subject" : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.patient = sbj
@@ -66,4 +66,3 @@ group createActDef
     "created"      : for src                 make tgt.created = evaluate( src, now() )
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-map-CommunicationRequest.fhirmap b/source/activitydefinition/activitydefinition-map-CommunicationRequest.fhirmap
index 855ee201007..5b17d966d24 100644
--- a/source/activitydefinition/activitydefinition-map-CommunicationRequest.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-CommunicationRequest.fhirmap
@@ -6,7 +6,7 @@ uses "http://hl7.org/fhir/StructureDefinition/CommunicationRequest" as target
 
 group main
     input src: Parameters as source
-    input tgt: ProcedureRequest as target
+    input tgt: CommunicationRequest as target
 
     "subject" : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.subject = sbj
@@ -36,7 +36,7 @@ endgroup
 
 group createActDef
     input src: ActivityDefinition as source
-    input tgt: ProcedureRequest as target
+    input tgt: CommunicationRequest as target
 
     "basedOn"      : for src                 make  tgt.basedOn = reference( src )
     "status"       : for src                 make  tgt.status = "draft"
@@ -51,4 +51,3 @@ group createActDef
 //    "ad.dosage"    : for src          as vvv check dosage.empty()
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-map-Contract.fhirmap b/source/activitydefinition/activitydefinition-map-Contract.fhirmap
index 9bfdebe54cc..1333e876e65 100644
--- a/source/activitydefinition/activitydefinition-map-Contract.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-Contract.fhirmap
@@ -6,7 +6,7 @@ uses "http://hl7.org/fhir/StructureDefinition/Contract" as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: Contract as target
 
     "subject" : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.subject = sbj
@@ -50,4 +50,3 @@ group createActDef
 //    "created"      : for src                 make tgt.created = evaluate( src, now() )
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-map-DeviceRequest.fhirmap b/source/activitydefinition/activitydefinition-map-DeviceRequest.fhirmap
index aa88a5d13b4..10e7343d7aa 100644
--- a/source/activitydefinition/activitydefinition-map-DeviceRequest.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-DeviceRequest.fhirmap
@@ -6,7 +6,7 @@ uses "http://hl7.org/fhir/StructureDefinition/DeviceRequest" as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: DeviceRequest as target
 
     "subject" : for src.parameter as p where name='subject' then {
             do: for p.value as sbj make tgt.subject = sbj
@@ -54,4 +54,3 @@ group createActDef
 //    "created"      : for src                 make tgt.created = evaluate( src, now() )
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-map-EnrollmentRequest.fhirmap b/source/activitydefinition/activitydefinition-map-EnrollmentRequest.fhirmap
index 7fd1ab66b63..82575d1583c 100644
--- a/source/activitydefinition/activitydefinition-map-EnrollmentRequest.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-EnrollmentRequest.fhirmap
@@ -6,7 +6,7 @@ uses "http://hl7.org/fhir/StructureDefinition/EnrollmentRequest" as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: EnrollmentRequest as target
 
     "subject" : for src.parameter as p where name='subject' then {
             do: for p.value as sbj make tgt.candidate = sbj
@@ -54,4 +54,3 @@ group createActDef
     "created"      : for src                 make tgt.created = evaluate( src, now() )
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-map-ImmunizationRecommendation.fhirmap b/source/activitydefinition/activitydefinition-map-ImmunizationRecommendation.fhirmap
index 7b855537235..2f1069f9be0 100644
--- a/source/activitydefinition/activitydefinition-map-ImmunizationRecommendation.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-ImmunizationRecommendation.fhirmap
@@ -6,7 +6,7 @@ uses "http://hl7.org/fhir/StructureDefinition/ImmunizationRecommendation" as tar
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: ImmunizationRecommendation as target
 
     "subject" : for src.parameter as p where name='subject' then {
             do: for p.value as sbj make tgt.patient = sbj
@@ -50,4 +50,3 @@ group createActDef
     "created"      : for src                 make tgt.date = evaluate( src, now() )
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-map-MedicationRequest.fhirmap b/source/activitydefinition/activitydefinition-map-MedicationRequest.fhirmap
index 96386e3f173..5659969bf47 100644
--- a/source/activitydefinition/activitydefinition-map-MedicationRequest.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-MedicationRequest.fhirmap
@@ -6,7 +6,7 @@ uses "http://hl7.org/fhir/StructureDefinition/MedicationRequest" as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: MedicationRequest as target
 
     "subject" : for src.parameter as p where name='subject' then {
             do: for p.value as sbj make tgt.subject = sbj
@@ -61,4 +61,3 @@ group createActDef
     "created"      : for src                 make tgt.authoredOn = evaluate( src, now() )
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-map-NutritionOrder.fhirmap b/source/activitydefinition/activitydefinition-map-NutritionOrder.fhirmap
index 7b1d3e02445..48eb11d46b8 100644
--- a/source/activitydefinition/activitydefinition-map-NutritionOrder.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-NutritionOrder.fhirmap
@@ -6,7 +6,7 @@ uses "http://hl7.org/fhir/StructureDefinition/NutritionOrder" as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: NutritionOrder as target
 
     "subject" : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.patient = sbj
@@ -61,4 +61,3 @@ group createActDef
 
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-map-ServiceRequest.fhirmap b/source/activitydefinition/activitydefinition-map-ServiceRequest.fhirmap
index ab1bf61e175..c93625bec86 100644
--- a/source/activitydefinition/activitydefinition-map-ServiceRequest.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-ServiceRequest.fhirmap
@@ -6,7 +6,7 @@ uses "http://hl7.org/fhir/StructureDefinition/ServiceRequest" as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: ServiceRequest as target
 
     "subject" : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.subject = sbj
@@ -50,4 +50,3 @@ group createActDef
 //    "ad.dosage"    : for src          as vvv check dosage.empty()
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-map-Task.fhirmap b/source/activitydefinition/activitydefinition-map-Task.fhirmap
index e28c536d607..9a2961ab3b0 100644
--- a/source/activitydefinition/activitydefinition-map-Task.fhirmap
+++ b/source/activitydefinition/activitydefinition-map-Task.fhirmap
@@ -36,7 +36,7 @@ endgroup
 
 group createActDef
     input src: ActivityDefinition as source
-    input tgt: ProcedureRequest as target
+    input tgt: Task as target
 
 //    "basedOn"      : for src                 make tgt.basedOn = reference( src )
     refAd          : for src.url as can   make tgt.instantiatesCanonical = can
@@ -60,4 +60,3 @@ group createActDef
 //    }
 endgroup
 
-
diff --git a/source/activitydefinition/activitydefinition-notes.xml b/source/activitydefinition/activitydefinition-notes.xml
index 57a03c6ce9e..481bd9aa211 100644
--- a/source/activitydefinition/activitydefinition-notes.xml
+++ b/source/activitydefinition/activitydefinition-notes.xml
@@ -59,7 +59,7 @@ uses &quot;http://hl7.org/fhir/StructureDefinition/Appointment&quot; as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: Appointment as target
 
     &quot;subject&quot; : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.participant as participant then {
@@ -93,7 +93,7 @@ endgroup
 
 group createActDef
     input src: ActivityDefinition as source
-    input tgt: ServiceRequest as target
+    input tgt: Appointment as target
 
 //    &quot;refAd&quot;        : for src.url      as vvv make tgt.basedOn = vvv
 //    &quot;ad.code&quot;      : for src.code     as vvv make tgt.code = vvv          // could be serviceCategory or specialty --&gt; unclear
@@ -141,7 +141,7 @@ uses &quot;http://hl7.org/fhir/StructureDefinition/AppointmentResponse&quot; as
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: AppointmentResponse as target
 
 //    &quot;subject&quot; : for src.parameter as p where name='subject' then {
 //        do: for p.value as sbj make tgt.subject = sbj
@@ -173,7 +173,7 @@ endgroup
 
 group createActDef
     input src: ActivityDefinition as source
-    input tgt: ServiceRequest as target
+    input tgt: AppointmentResponse as target
 
 //    &quot;refAd&quot;        : for src.url      as vvv make tgt.instantiatesCanonical = vvv
     &quot;status&quot;       : for src                 make tgt.participantStatus = &quot;needs-action&quot;
@@ -198,7 +198,7 @@ uses &quot;http://hl7.org/fhir/StructureDefinition/CarePlan&quot; as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: CarePlan as target
 
     &quot;subject&quot; : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.subject = sbj
@@ -228,7 +228,7 @@ endgroup
 
 group createActDef
     input src: ActivityDefinition as source
-    input tgt: ServiceRequest as target
+    input tgt: CarePlan as target
 
     &quot;ad.code&quot;      : for src.code     as vvv  make tgt.category = vvv
 //    &quot;ad.priority&quot;  : for src.priority as prio make tgt.priority = vvv
@@ -257,7 +257,7 @@ uses &quot;http://hl7.org/fhir/StructureDefinition/Claim&quot; as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: Claim as target
 
     &quot;subject&quot; : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.patient = sbj
@@ -328,7 +328,7 @@ uses &quot;http://hl7.org/fhir/StructureDefinition/CommunicationRequest&quot; as
 
 group main
     input src: Parameters as source
-    input tgt: ProcedureRequest as target
+    input tgt: CommunicationRequest as target
 
     &quot;subject&quot; : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.subject = sbj
@@ -358,7 +358,7 @@ endgroup
 
 group createActDef
     input src: ActivityDefinition as source
-    input tgt: ProcedureRequest as target
+    input tgt: CommunicationRequest as target
 
     &quot;basedOn&quot;      : for src                 make  tgt.basedOn = reference( src )
     &quot;status&quot;       : for src                 make  tgt.status = &quot;draft&quot;
@@ -384,7 +384,7 @@ uses &quot;http://hl7.org/fhir/StructureDefinition/Contract&quot; as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: Contract as target
 
     &quot;subject&quot; : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.subject = sbj
@@ -439,7 +439,7 @@ uses &quot;http://hl7.org/fhir/StructureDefinition/DeviceRequest&quot; as target
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: DeviceRequest as target
 
     &quot;subject&quot; : for src.parameter as p where name='subject' then {
             do: for p.value as sbj make tgt.subject = sbj
@@ -498,7 +498,7 @@ uses &quot;http://hl7.org/fhir/StructureDefinition/EnrollmentRequest&quot; as ta
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: EnrollmentRequest as target
 
     &quot;subject&quot; : for src.parameter as p where name='subject' then {
             do: for p.value as sbj make tgt.candidate = sbj
@@ -557,7 +557,7 @@ uses &quot;http://hl7.org/fhir/StructureDefinition/ImmunizationRecommendation&qu
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: ImmunizationRecommendation as target
 
     &quot;subject&quot; : for src.parameter as p where name='subject' then {
             do: for p.value as sbj make tgt.patient = sbj
@@ -612,7 +612,7 @@ uses &quot;http://hl7.org/fhir/StructureDefinition/MedicationRequest&quot; as ta
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: MedicationRequest as target
 
     &quot;subject&quot; : for src.parameter as p where name='subject' then {
             do: for p.value as sbj make tgt.subject = sbj
@@ -678,7 +678,7 @@ uses &quot;http://hl7.org/fhir/StructureDefinition/NutritionOrder&quot; as targe
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: NutritionOrder as target
 
     &quot;subject&quot; : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.patient = sbj
@@ -744,7 +744,7 @@ uses &quot;http://hl7.org/fhir/StructureDefinition/ServiceRequest&quot; as targe
 
 group main
     input src: Parameters as source
-    input tgt: ReferralRequest as target
+    input tgt: ServiceRequest as target
 
     &quot;subject&quot; : for src.parameter as p where name='subject' then {
         do: for p.value as sbj make tgt.subject = sbj
@@ -886,7 +886,7 @@ endgroup
 
 group createActDef
     input src: ActivityDefinition as source
-    input tgt: ProcedureRequest as target
+    input tgt: Task as target
 
 //    &quot;basedOn&quot;      : for src                 make tgt.basedOn = reference( src )
     refAd          : for src.url as can   make tgt.instantiatesCanonical = can
