diff --git a/source/library/library-fhir-helpers-content.cql b/source/library/library-fhir-helpers-content.cql
index 3b9470f2983..0f057c781e8 100644
--- a/source/library/library-fhir-helpers-content.cql
+++ b/source/library/library-fhir-helpers-content.cql
@@ -136,7 +136,7 @@ define function ToString(value FHIR.RelatedArtifactType): value.value
 define function ToString(value FHIR.oid): value.value
 define function ToString(value FHIR.CompartmentType): value.value
 define function ToString(value FHIR.medicationrequestStatus): value.value
-define function ToString(value FHIR.InvoicePriceComponentType): value.value
+define function ToString(value FHIR.PriceComponentType): value.value
 define function ToString(value FHIR.DeviceMetricCalibrationState): value.value
 define function ToString(value FHIR.GroupType): value.value
 define function ToString(value FHIR.EnableWhenBehavior): value.value
