diff --git a/images/mapping.g4 b/images/mapping.g4
index c2b3cc0ed1b..b97b3281d7f 100644
--- a/images/mapping.g4
+++ b/images/mapping.g4
@@ -88,7 +88,7 @@ parameter
   ;
 
 mapRules
-  : '{' mapRule* '}'
+  : '{' mapIdentityCopy? mapRule* '}'
   ;
 
 typeMode
@@ -108,6 +108,10 @@ mapRule
   | mapTransformationRule ';'                                 #mapFhirMarkup
  	;
 
+mapIdentityCopy
+  : qualifiedIdentifier '->' qualifiedIdentifier ':' qualifiedIdentifier (',' qualifiedIdentifier)* ';'
+  ;
+
 mapTransformationRule
   : ruleSources ('->' ruleTargets)? dependentExpression? ruleName?
   ;
