diff --git a/source/fhir.ini b/source/fhir.ini
index 76dfefc1f22..bd19256931d 100644
--- a/source/fhir.ini
+++ b/source/fhir.ini
@@ -2391,6 +2391,7 @@ Measure.supplementalData.valueSet=+
 Measure.type=
 MeasureReport.category=+
 MeasureReport.group.calculatedDate=+
+MeasureReport.group.compositeScoring=+
 MeasureReport.group.description=+
 MeasureReport.group.improvementNotation=+
 MeasureReport.group.improvementNotationGuidance=+
