diff --git a/source/oids.ini b/source/oids.ini
index 8ba5c860047..3a6f20a257a 100644
--- a/source/oids.ini
+++ b/source/oids.ini
@@ -990,7 +990,7 @@ http://hl7.org/fhir/valueset/reason-medication-not-taken-codes = 2.16.840.1.1138
 http://terminology.hl7.org/ValueSet/gender-identity = 2.16.840.1.113883.4.642.3.3291
 http://terminology.hl7.org/ValueSet/pronouns = 2.16.840.1.113883.4.642.3.3290
 http://terminology.hl7.org/ValueSet/recorded-sex-or-gender-type = 2.16.840.1.113883.4.642.3.3289
-http://terminology.hl7.org/ValueSet/sex-for-clinical-use = 2.16.840.1.113883.4.642.3.3181
+http://terminology.hl7.org/ValueSet/sex-parameter-for-clinical-use = 2.16.840.1.113883.4.642.3.3181
 http://hl7.org/fhir/ValueSet/value-filter-comparator = 2.16.840.1.113883.4.642.3.3306
 http://hl7.org/fhir/ValueSet/device-category = 2.16.840.1.113883.4.642.3.3307
 http://hl7.org/fhir/ValueSet/device-operationstatus = 2.16.840.1.113883.4.642.3.3308
@@ -1613,7 +1613,7 @@ http://hl7.org/fhir/research-study-statusDate-activity = 2.16.840.1.113883.4.642
 http://hl7.org/fhir/restful-security-service = 2.16.840.1.113883.4.642.4.1079
 ;http://terminology.hl7.org/CodeSystem/service-provision-conditions = 2.16.840.1.113883.4.642.4.1143
 ;http://terminology.hl7.org/CodeSystem/service-referral-method = 2.16.840.1.113883.4.642.4.1142
-;http://terminology.hl7.org/CodeSystem/sex-for-clinical-use = 2.16.840.1.113883.4.642.4.2038
+;http://terminology.hl7.org/CodeSystem/sex-parameter-for-clinical-use = 2.16.840.1.113883.4.642.4.2038
 ;http://terminology.hl7.org/CodeSystem/smart-capabilities = 2.16.840.1.113883.4.642.1.1265
 ;http://terminology.hl7.org/CodeSystem/special-values = 2.16.840.1.113883.4.642.1.1049
 ;http://terminology.hl7.org/CodeSystem/state-change-reason = 2.16.840.1.113883.4.642.4.2009
