diff --git a/images/subscriptions/subscription-error-missing-event.plantuml b/images/subscriptions/subscription-error-missing-event.plantuml
index 0e4571b1571..a3418a4fadd 100644
--- a/images/subscriptions/subscription-error-missing-event.plantuml
+++ b/images/subscriptions/subscription-error-missing-event.plantuml
@@ -29,7 +29,7 @@ group Subscriber Recovery Process
 end
 
 group Re-enable Subscription
-  C -> S:Update Subscription\n""status: requested""
+  C -> S:Update Subscription\n""Subscription.status: requested""
   return ""200: OK""
   ...
 end
diff --git a/images/subscriptions/subscription-error-no-communication.plantuml b/images/subscriptions/subscription-error-no-communication.plantuml
index 70f8ddb98f7..2039bed60b2 100644
--- a/images/subscriptions/subscription-error-no-communication.plantuml
+++ b/images/subscriptions/subscription-error-no-communication.plantuml
@@ -35,7 +35,7 @@ group Recovery Process
 end
 
 group Re-enable Subscription
-  C -> S:Update Subscription\n""status: requested""
+  C -> S:Update Subscription\n""Subscription.status: requested""
   return ""200: OK""
   ...
 end
diff --git a/images/subscriptions/subscription-workflow-general.plantuml b/images/subscriptions/subscription-workflow-general.plantuml
index be5a9788560..afe19282d0d 100644
--- a/images/subscriptions/subscription-workflow-general.plantuml
+++ b/images/subscriptions/subscription-workflow-general.plantuml
@@ -33,7 +33,7 @@ else Valid Subscription, without Handshake
   note right of S: Send notifications...
   ...
 else Valid Subscription, with Handshake Success
-  S -->> C: Success: status = ""requested""
+  S -->> C: Success: ""Subscription.status = requested""
   S ->> E: Handshake
   E -->> S: Success, e.g.:\n""200: OK""
   note right of S
