diff --git a/source/fhir.ini b/source/fhir.ini
index e97e3e3636f..76238a566bc 100644
--- a/source/fhir.ini
+++ b/source/fhir.ini
@@ -2578,15 +2578,15 @@ SubscriptionTopic.canFilterBy.filterParameter=@SubscriptionTopic.trigger.canFilt
 SubscriptionTopic.canFilterBy.modifier=@SubscriptionTopic.trigger.canFilterBy.modifier
 SubscriptionTopic.canFilterBy.resource=@SubscriptionTopic.trigger.canFilterBy.resource
 SubscriptionTopic.canFilterBy=@SubscriptionTopic.trigger.canFilterBy
-SubscriptionTopic.eventTrigger.description=
-SubscriptionTopic.eventTrigger.event=
-SubscriptionTopic.eventTrigger.resource=
-SubscriptionTopic.eventTrigger=
-SubscriptionTopic.notificationShape.include=
+SubscriptionTopic.eventTrigger.description=@SubscriptionTopic.trigger.description
+SubscriptionTopic.eventTrigger.event=@SubscriptionTopic.trigger.event
+SubscriptionTopic.eventTrigger.resource=@SubscriptionTopic.trigger.resource
+SubscriptionTopic.eventTrigger=@SubscriptionTopic.trigger
+SubscriptionTopic.notificationShape.include=@SubscriptionTopic.trigger.notificationShape.include
 SubscriptionTopic.notificationShape.resource=@SubscriptionTopic.trigger.notificationShape.resource
 SubscriptionTopic.notificationShape.revInclude=@SubscriptionTopic.trigger.notificationShape.revInclude
 SubscriptionTopic.notificationShape=@SubscriptionTopic.trigger.notificationShape
-SubscriptionTopic.resourceTrigger.description=
+SubscriptionTopic.resourceTrigger.description=@SubscriptionTopic.trigger.description
 SubscriptionTopic.resourceTrigger.fhirPathCriteria=@SubscriptionTopic.trigger.fhirPathCriteria
 SubscriptionTopic.resourceTrigger.queryCriteria.current=@SubscriptionTopic.trigger.queryCriteria.current
 SubscriptionTopic.resourceTrigger.queryCriteria.previous=@SubscriptionTopic.trigger.queryCriteria.previous
@@ -2594,8 +2594,8 @@ SubscriptionTopic.resourceTrigger.queryCriteria.requireBoth=@SubscriptionTopic.t
 SubscriptionTopic.resourceTrigger.queryCriteria.resultForCreate=@SubscriptionTopic.trigger.queryCriteria.resultForCreate
 SubscriptionTopic.resourceTrigger.queryCriteria.resultForDelete=@SubscriptionTopic.trigger.queryCriteria.resultForDelete
 SubscriptionTopic.resourceTrigger.queryCriteria=@SubscriptionTopic.trigger.queryCriteria
-#SubscriptionTopic.resourceTrigger.resource=
-#SubscriptionTopic.resourceTrigger.supportedInteraction=
+SubscriptionTopic.resourceTrigger.resource=@SubscriptionTopic.trigger.resource
+SubscriptionTopic.resourceTrigger.supportedInteraction=@SubscriptionTopic.trigger.supportedInteraction
 SubscriptionTopic.resourceTrigger=@SubscriptionTopic.trigger
 #SubscriptionTopic.trigger.canFilterBy.comparator=+
 #SubscriptionTopic.trigger.canFilterBy.description=+
