diff --git a/source/bindings.ini b/source/bindings.ini
index e8c4905cfac..e0c0e554fca 100644
--- a/source/bindings.ini
+++ b/source/bindings.ini
@@ -121,11 +121,11 @@ ProvenanceParticipantRole = 112
 AuditEventSeverity = 114
 AuditEventEventAction = 115
 AuditEventEventOutcome = 116
-AuditEventParticipantNetworkType = 117
+AuditEventAgentNetworkType = 117
 AuditEventSourceType = 118
-AuditEventObjectType = 119
-AuditEventObjectRole = 120
-AuditEventObjectLifecycle = 121
+AuditEventEntityType = 119
+AuditEventEntityRole = 120
+AuditEventEntityLifecycle = 121
 AuditEventObjectIdType = 122
 FHIRContentType = 123
 IdentifierUse = 124
