diff --git a/source/oids.ini b/source/oids.ini
index 8947b13924f..f7697217b17 100644
--- a/source/oids.ini
+++ b/source/oids.ini
@@ -1907,7 +1907,6 @@ http://hl7.org/fhir/StructureDefinition/cqm-ValidityPeriod = 2.16.840.1.113883.4
 http://hl7.org/fhir/StructureDefinition/derivation-reference = 2.16.840.1.113883.4.642.5.1425
 http://hl7.org/fhir/StructureDefinition/device-commercialBrand = 2.16.840.1.113883.4.642.5.1381
 http://hl7.org/fhir/StructureDefinition/device-implantStatus = 2.16.840.1.113883.4.642.5.1382
-http://hl7.org/fhir/StructureDefinition/devicemetricobservation = 2.16.840.1.113883.4.642.5.1476
 http://hl7.org/fhir/StructureDefinition/devicerequest-patientInstruction = 2.16.840.1.113883.4.642.5.1094
 http://hl7.org/fhir/StructureDefinition/diagnosticReport-addendumOf = 2.16.840.1.113883.4.642.5.1391
 http://hl7.org/fhir/StructureDefinition/diagnosticReport-extends = 2.16.840.1.113883.4.642.5.1390
