diff --git a/publish.ini b/publish.ini
index eb34d8c1033..e9723ed708a 100644
--- a/publish.ini
+++ b/publish.ini
@@ -474,7 +474,6 @@ Terminology Service = terminology-service.html
 [redirects]
 http://hl7.org/fhir/canonicalization/xml = Canonical XML;xml.html
 http://hl7.org/fhir/canonicalization/json = Canonical JSON;json.html
-http://hl7.org/fhir/ValueSet/@all = Magic ValueSet Id;terminology-service.html#all
 http://hl7.org/fhir/nil = Magic RDF nil value;rdf.html#nil
 
 [sid]
