diff --git a/source/consent/consent-example-notSecLabel.xml b/source/consent/consent-example-notSecLabel.xml
index 9a8d8e285bc..363af0c69cf 100644
--- a/source/consent/consent-example-notSecLabel.xml
+++ b/source/consent/consent-example-notSecLabel.xml
@@ -57,7 +57,7 @@
       </coding>
     </action>
     <securityLabel>
-      <system value="http://terminology.hl7.org/ValueSet/v3-InformationSensitivityPolicy" />
+      <system value="http://terminology.hl7.org/CodeSystem/v3-ActCode" />
       <code value="HIV" />
     </securityLabel>
   </provision>
diff --git a/source/group/group-example-family.xml b/source/group/group-example-family.xml
index 077ed98a754..4fac88a13a3 100644
--- a/source/group/group-example-family.xml
+++ b/source/group/group-example-family.xml
@@ -17,7 +17,7 @@ compared to real world language usage
   <membership value="enumerated"/>
   <code>
     <coding>
-      <system value="http://hl7.org/fhir/ValueSet/group-code"/>
+      <system value="http://hl7.org/fhir/group-code"/>
       <code value="family"/>
     </coding>
   </code>
diff --git a/source/group/group-example-household.xml b/source/group/group-example-household.xml
index f09bc298d41..816ced27eb7 100644
--- a/source/group/group-example-household.xml
+++ b/source/group/group-example-household.xml
@@ -17,7 +17,7 @@ compared to real world language usage
   <membership value="enumerated"/>
   <code>
     <coding>
-      <system value="http://hl7.org/fhir/ValueSet/group-code"/>
+      <system value="http://hl7.org/fhir/group-code"/>
       <code value="household"/>
     </coding>
   </code>
diff --git a/source/library/library-hiv-indicators-content.cql b/source/library/library-hiv-indicators-content.cql
index 2458f424072..66700599299 100644
--- a/source/library/library-hiv-indicators-content.cql
+++ b/source/library/library-hiv-indicators-content.cql
@@ -16,7 +16,7 @@ codesystem "SNOMED-CT": 'http://snomed.info/sct/900000000000207008'
 // LOINC, 2.63+
 codesystem "LOINC": 'http://loinc.org'
 
-codesystem "ISO-8601-Derived Periods": 'http://ohie.org/ValueSet/iso-8601-derived-periods' // { 'P0Y--P1Y', 'P1Y--P5Y', ... }
+codesystem "Time Period Ranges": 'http://terminology.hl7.org/CodeSystem/time-period-ranges' // { 'P0Y--P1Y', 'P1Y--P5Y', ... }
 codesystem "PMTCT HIV Status Codes": 'TBD' // { 'known-positive', 'newly-identified-positive', 'newly-identified-negative' }
 codesystem "PMTCT ART Status Codes": 'TBD' // { 'already-on-art', 'new-on-art' }
 
@@ -43,17 +43,17 @@ code "HIV 1 and 2 tests - Meaningful Use set": '75622-1' from "LOINC" display 'H
 code "History of antiretroviral therapy (situation)": '432101000124108' from "SNOMED-CT" display 'History of antiretroviral therapy (situation)'
 
 // Age Groups
-code "P0Y--P1Y": 'P0Y--P1Y' from "ISO-8601-Derived Periods" display '< 1 year'
-code "P1Y--P5Y": 'P1Y--P5Y' from "ISO-8601-Derived Periods" display '1-4 years'
-code "P5Y--P10Y": 'P5Y--P10Y' from "ISO-8601-Derived Periods" display '5-9 year'
-code "P10Y--P15Y": 'P10Y--P15Y' from "ISO-8601-Derived Periods" display '10-14 year'
-code "P15Y--P20Y": 'P15Y--P20Y' from "ISO-8601-Derived Periods" display '15-19 year'
-code "P20Y--P25Y": 'P20Y--P25Y' from "ISO-8601-Derived Periods" display '20-24 year'
-code "P25Y--P30Y": 'P25Y--P30Y' from "ISO-8601-Derived Periods" display '25-29 year'
-code "P30Y--P35Y": 'P30Y--P35Y' from "ISO-8601-Derived Periods" display '30-34 year'
-code "P35Y--P40Y": 'P35Y--P40Y' from "ISO-8601-Derived Periods" display '35-39 year'
-code "P40Y--P50Y": 'P40Y--P50Y' from "ISO-8601-Derived Periods" display '40-49 year'
-code "P50Y--P9999Y": 'P50Y--P9999Y' from "ISO-8601-Derived Periods" display '50+ years'
+code "P0Y--P1Y": 'P0Y--P1Y' from "Time Period Ranges" display '< 1 year'
+code "P1Y--P5Y": 'P1Y--P5Y' from "Time Period Ranges" display '1-4 years'
+code "P5Y--P10Y": 'P5Y--P10Y' from "Time Period Ranges" display '5-9 year'
+code "P10Y--P15Y": 'P10Y--P15Y' from "Time Period Ranges" display '10-14 year'
+code "P15Y--P20Y": 'P15Y--P20Y' from "Time Period Ranges" display '15-19 year'
+code "P20Y--P25Y": 'P20Y--P25Y' from "Time Period Ranges" display '20-24 year'
+code "P25Y--P30Y": 'P25Y--P30Y' from "Time Period Ranges" display '25-29 year'
+code "P30Y--P35Y": 'P30Y--P35Y' from "Time Period Ranges" display '30-34 year'
+code "P35Y--P40Y": 'P35Y--P40Y' from "Time Period Ranges" display '35-39 year'
+code "P40Y--P50Y": 'P40Y--P50Y' from "Time Period Ranges" display '40-49 year'
+code "P50Y--P9999Y": 'P50Y--P9999Y' from "Time Period Ranges" display '50+ years'
 
 // PMTCT ART status
 code "Already on ART": 'already-on-art' from "PMTCT ART Status Codes" display 'Already on ART'
diff --git a/source/measure/valueset-measure-stratifier-age-range-example.xml b/source/measure/valueset-measure-stratifier-age-range-example.xml
index a195393be86..42293dc1c5e 100644
--- a/source/measure/valueset-measure-stratifier-age-range-example.xml
+++ b/source/measure/valueset-measure-stratifier-age-range-example.xml
@@ -18,7 +18,7 @@
   <description value="An example value set representing a possible age range stratification."/>
   <compose>
     <include>
-      <system value="http://ohie.org/ValueSet/iso-8601-derived-periods"/>
+      <system value="http://terminology.hl7.org/CodeSystem/time-period-ranges"/>
       <concept><code value="P0Y--P1Y"/> <display value="&lt; 1 year"/></concept>
       <concept><code value="P1Y--P5Y"/> <display value="1-4 years"/></concept>
       <concept><code value="P5Y--P10Y"/> <display value="5-9 years"/></concept>
