diff --git a/source/fhir.ini b/source/fhir.ini
index 3182067d00..ac1c39cb14 100644
--- a/source/fhir.ini
+++ b/source/fhir.ini
@@ -2499,11 +2499,11 @@ RelatedArtifact.publicationDate=
 RelatedArtifact.publicationStatus=
 Requirements.actor.key=+
 Requirements.actor.reference=+
-Requirements.category=+
 Requirements.imports.key=+
 Requirements.imports.reference=+
 Requirements.imports=+
 Requirements.statement.actor=+
+Requirements.statement.category=+
 Requirements.statement.derivedFrom.key=+
 Requirements.statement.derivedFrom.reference=+
 Requirements.statement.parent=-> partOf - though not directly mappable
