diff --git a/source/fhir.ini b/source/fhir.ini
index 56521cb77f..a9e6522f40 100644
--- a/source/fhir.ini
+++ b/source/fhir.ini
@@ -2120,6 +2120,7 @@ Claim.item.subject=+
 Claim.patient=@Claim.subject
 Claim.statusReason=+
 Claim.supportingInfo.subCategory=+
+ClaimResponse.item.adjudication.decisionDate=+
 ClaimResponse.patient=@ClaimResponse.subject
 ClinicalUseDefinition.contraindication.management=+
 ClinicalUseDefinition.contraindication.otherTherapy.relationshipType=
