diff --git a/source/fhir.ini b/source/fhir.ini
index 76238a566b..97f0145cee 100644
--- a/source/fhir.ini
+++ b/source/fhir.ini
@@ -1281,7 +1281,7 @@ fhir-old-types=terminologies\codesystem-fhir-old-types.xml
 color-names=terminologies\codesystem-color-names.xml
 color-rgb=terminologies\codesystem-color-rgb.xml
 simple-format-codes=terminologies\codesystem-simple-format-codes.xml
-;patch-operation=parameters\codesystem-patch-operation.xml
+patch-operation=parameters\codesystem-patch-operation.xml
 
 [valuesets]
 ;vitalsignresult=observation\valueset-observation-vitalsignresult.xml
@@ -1308,7 +1308,7 @@ mimetypes=terminologies\valueset-mimetypes.xml
 supplemented-mimetypes=terminologies\valueset-supplemented-mimetypes.xml
 example-characteristic-codes=group\valueset-example-characteristic-codes.xml
 item-type=questionnaire\valueset-item-type.xml
-;patch-operation=parameters\valueset-patch-operation.xml
+patch-operation=parameters\valueset-patch-operation.xml
 search-modifier-code=searchparameter\valueset-search-modifier-code.xml
 
 [valueset-fixup]
